src/defaults.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+const { defaultOptions } = require("./utils");
+const { sign } = require("tweetnacl");
+const { toByteArray } = require("base64-js");
+
+BigInt.prototype.toJSON = function () {
+  return this.toString();
+};
+
+const defaultBaseOptions = {
+  APIKey: "",
+  skynetApiKey: "",
+  customUserAgent: "",
+  customCookie: "",
+  onDownloadProgress: undefined,
+  onUploadProgress: undefined,
+  loginFn: undefined,
+};
+
+const defaultGetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointGetEntry: "/skynet/registry",
+  hashedDataKeyHex: false,
+};
+
+const defaultSetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointSetEntry: "/skynet/registry",
+  hashedDataKeyHex: false,
+};
+
+const defaultSkydbOptions = {
+  ...defaultOptions("/skynet/skyfile"),
+  portalFileFieldname: "file",
+};
+
+const json_response_version = 2;
+
+/**
+ * Sets the hidden _data and _v fields on the given raw JSON data.
+ *
+ * @param data - The given JSON data.
+ * @returns - The Skynet JSON data.
+ */
+const buildSkynetJsonObject = async function (data) {
```

Comment:
```suggestion
const buildSkynetJsonObject = function (data) {
```

Reply:
can i change
but maybe later we can import the function directly from "skynet-js". 
Let's see if my request goes through with "skynet-js".

look here:
https://github.com/SkynetLabs/skynet-js/issues/468